source/_integrations/input_datetime.markdown

```diff
@@ -16,9 +16,9 @@ The `input_datetime` integration allows the user to define date and time values
 that can be controlled via the frontend and can be used within automations and
 templates.
 
-The preferred way to configure input datetime is via the user interface at **Settings** -> **Devices & Services** -> **Helpers**. Click the add button and then choose the **Date and/or time** option.
+The preferred way to configure input datetime is via the user interface at **{% my helpers title="Settings -> Devices & Services -> Helpers" %}**. Click the add button and then choose the **{% my config_flow_start domain=input_datetime title="Date and/or time" %}** option.
```

Comment:
```suggestion
The preferred way to configure input datetime is via the user interface at **{% my helpers title="Settings > Devices & Services > Helpers" %}**. Click the add button and then choose the **{% my config_flow_start domain=input_datetime title="Date and/or time" %}** option.
```
I know you did not make these. But while you're at it, let's change them. We're trying to us just these > now.

Reply:
I'm not going to change the "->" because these are almost everywhere in the documentation and changing them would require a larger PR.

Reply:
Sorry, that makes no sense to me. We are already touching right now, why not just do it?

../Frenck